### PR TITLE
update gh-pages

### PR DESCRIPTION
**Pull request by jongund**
@mpnkhan 
I fixed some documentation issues with the gh-pages, please merge,

### CODE REVIEW

**Comment by brianteeman** [comment]
I will check everything by the weekend and merge it and make a new release on npm. I have a few other small fixes to include.

File: _layouts/index.html [hunk 1]
```diff
@@ -18,12 +18,6 @@
 						alt="Skip To Logo"></a>
 				<h1 id="project_title" style="display:inline; margin-left:10px;position:relative;top:-100px;">SkipTo
 					plugin by PayPal Accessibility Team</h1>
-				<section id="downloads">
-					<a class="zip_download_link" href="https://github.com/paypal/skipto/zipball/master">Download this
-						project as a .zip file</a>
-					<a class="tar_download_link" href="https://github.com/paypal/skipto/tarball/master">Download this
-						project as a tar.gz file</a>
-				</section>
```
Comment (brianteeman) [inline _layouts/index.html:26]: your call but I would prefer to fix the links which means making a release here on github which I can do

Comment (jongund) [reply _layouts/index.html:26]: I am not sure how these files are generated, but if the export the most current version, we can leave them in.

Comment (brianteeman) [reply _layouts/index.html:26]: I _think_ those links are to releases on github. I guess we will see when I build it. If not then I will remove the links

Comment (brianteeman) [reply _layouts/index.html:26]: I changed the link and these work now

**Comment by brianteeman** [comment]
the downloads source folder was done automatically and the download links are now working so I created a new pr with just the changes to the index.md from here

**Pull request closed without merging**